.github/workflows/ami-release-nix.yml

```diff
@@ -3,17 +3,37 @@ name: Release AMI Nix
 on:
   push:
     branches:
-      - develop
-      - release/*
+      #- develop
+      #- release/*
+      - sam/15-16-ghactions
```

Comment:
Needs reverting?

Reply:
This is a pr to another branch where that is commented out. My intention is to merge this to that branch, once that branch pr is approved I would revert this change to trigger release

Reply:
@soedirgo okto merge to parent pr?

Reply:
Yeah parent PR looks OK, but need a +1 from infra